### ISSUE TITLE
Is this still working

### ISSUE THREAD

**Issue by mssmison** [emcniece/hass-bchydro#7]
Is this still working/in development? I tried installing the custom component tonight but the sensors are just showing as unavailable.

**Comment by emcniece** [comment]
Yes, apologies... I've been stuck on a refactor in https://github.com/emcniece/hass-bchydro/pull/6 for a while. It's on my list 👍

**Comment by mssmison** [comment]
No need to apologise!

Glad to great it's still on the list.

Cheers
